test/image/mocks/marker_symbols.json

```diff
@@ -0,0 +1,99 @@
+{
+  "data": [
+    {
+      "x": [1, 2, 3, 4, 5, 6, 7, 8, 9, 1, 2, 3, 4, 5, 6, 7, 8, 9, 1, 2, 3, 4, 5, 6, 7, 8, 9, 1, 2, 3, 4, 5, 6, 7, 8, 9, 1, 2, 3, 4, 5, 6, 7, 8, 9],
+      "y": [1, 1, 1, 1, 1, 1, 1, 1, 1, 2, 2, 2, 2, 2, 2, 2, 2, 2, 3, 3, 3, 3, 3, 3, 3, 3, 3, 4, 4, 4, 4, 4, 4, 4, 4, 4, 5, 5, 5, 5, 5, 5, 5, 5, 5],
+      "mode": "markers",
+      "text": [
+        "marker number: 0<br>marker string: 'circle'",
```

Comment:
@alexcjohnson Turns out we didn't have a mock testing _all_ our marker symbols.

I dug out https://github.com/plotly/streambed/pull/358 and https://github.com/plotly/documentation/pull/16 and made this one here.